fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1116,6 +1142,38 @@ ACTOR Future<Void> applyMetadataToCommittedTransactions(CommitBatchContext* self
 	return Void();
 }
 
+void writeMutation(CommitBatchContext* self, int64_t tenantId, MutationRef mutation) {
+	if (!SERVER_KNOBS->ENABLE_COMMIT_PROXY_ENCRYPTION) {
+		self->toCommit.writeTypedMessage(mutation);
+		return;
+	}
+
+	// TODO: Is this correct?
```

Comment:
sounds correct to me..

Reply:
yeah, I've changed ENCRYPT_INVALID_DOMAIN_ID to TENANT_INVALID_ID, which is -1.

Reply:
what I'm not sure is is it expected to get TENANT_INVALID_ID as tenant id here. I saw it happen in simulation, but didn't look what that test does.